cluster/addons/calico-policy-controller/calico-clusterrole.yaml

```diff
@@ -19,6 +19,13 @@ rules:
       - endpoints
     verbs:
       - get
+  - apiGroups: [""]
```

Comment:
You could just add this right below `namespaces` above since they're the same apigroup and verbs. 

Not a big deal though.

Reply:
Good point! Just follow your suggestions! Less change is always preferred.